scene/2d/audio_stream_player_2d.h

```diff
@@ -35,6 +35,7 @@
 #include "scene/2d/node_2d.h"
 #include "servers/audio/audio_stream.h"
 #include "servers/audio_server.h"
+#include <cstdint>
```

Comment:
```suggestion
#include "scene/2d/node_2d.h"
#include "servers/audio/audio_stream.h"
#include "servers/audio_server.h"

#include <cstdint>
```

See https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

I don't know if the explicit `<cstdint>` include is needed though, technically it's correct but we use `uint*_t` types throughout the codebase without explicit includes, so I guess it's already included via typedefs which are included basically everywhere.